hawkular-alerts-rest/hawkular-alerts-rest-api/src/main/java/org/hawkular/alerts/rest/AlertsHandler.java

```diff
@@ -210,16 +221,9 @@ We maintain old tags criteria as deprecated (it can be removed in a next major v
             @ApiResponse(code = 400, message = "Bad Request/Invalid Parameters.", response = ApiError.class)
     })
     public Response ackAlert(
-            @ApiParam(required = true, value = "The alertId to Ack.",
-                allowableValues = "An existing alertId.")
-            @PathParam("alertId")
-            final String alertId,
-            @ApiParam(required = false, value = "User acknowledging the alerts.")
-            @QueryParam("ackBy")
-            final String ackBy,
-            @ApiParam(required = false, value = "Additional notes associated with the acknowledgement.")
-            @QueryParam("ackNotes")
-            final String ackNotes) {
+            @ApiParam(required = true, value = "The alertId to Ack.", allowableValues = "An existing alertId.") @PathParam("alertId") final String alertId,
```

Comment:
Let's not break/change the format here.
I think each annotation should go in a different line for consistency and readability.
Perhaps some automatic formatting from the IDE here.